feature/rds

ana0 opened the pull request.

xwvvvvwx commented on circles-api/modules/rds/main.tf:
```diff
@@ -0,0 +1,70 @@
+  identifier = "${var.rds_instance_identifier}"
+  allocated_storage = "${var.allocated_storage}"
+  engine = "postgres"
+  engine_version = "9.6.3"
```
why not use the latest version of postgres?

xwvvvvwx commented on circles-api/modules/rds/main.tf:
```diff
@@ -0,0 +1,70 @@
+
+resource "aws_db_subnet_group" "default" {
+  name = "${var.rds_instance_identifier}-subnet-group"
+  description = "Terraform example RDS subnet group"
```
copy paste error.

xwvvvvwx commented on circles-api/notes.txt:
```diff
@@ -0,0 +1,11 @@
+the terraform state goes in s3
```
do we wanna commit this file?

ana0 commented:
Addressed PR comments, except for deletion_protection .. looks like it was added to terraform only really recently (it's not in my version yet, for example) .. https://github.com/terraform-providers/terraform-provider-aws/issues/6008

Do we all want to upgrade?

xwvvvvwx commented:
I'm fine with requiring the latest version of terraform.

edzillion commented:
Problems `apply`ing this due to problems with CIDR block allocation.
